pyaedt/modeler/geometry_operators.py

```diff
@@ -765,8 +765,8 @@ def pointing_to_axis(x_pointing, y_pointing):
     @staticmethod
     @pyaedt_function_handler()
     def axis_to_euler_zxz(x, y, z):
-        """Retrieve the Euler angles of a frame defined by X, Y, and Z axes, following the rotation sequence ZXZ.
-
+        """Retrieve the Euler angles of a frame defined by X, Y, and Z axes, following the rotation
+        sequence ZXZ.
         Provides assumption for the gimbal lock problem.
```

Comment:
```suggestion
        """Retrieve Euler angles of a frame defined by X, Y, and Z axes, following rotation sequence ZXZ.
        
        This method provides assumption for the gimbal lock problem.
```
The edit is an attempt to get the description of the method to fit on one line. I question the use of "assumption" in "This method provides assumption ...." Also, should gimbal have an initial capital letter because it's someone's name?

Reply:
Here @Alberto-DM can help us. I just fixed the logic inside the function. @Alberto-DM What do you think? Maybe this part is not even needed?
`
         The frame is defined by X, Y, and Z axes. This method provides assumption
         for the gimbal lock problem.
`